wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Reaction/Handler.js

```diff
@@ -23,6 +23,8 @@ define(
 	{
 		"use strict";
 		
+		var UiScreen = require('Ui/Screen');
```

Comment:
Why is this not an explicit dependency within the definition? Add a comment!

Reply:
I'm not even sure why it won't work as a regular import. I suspected a circular dependency, but I didn't found one.

Reply:
I just tested the following on a desktop chrome:

```
From fc17fd1171d4ad9ee59996fbb1d5257fea353086 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tim=20D=C3=BCsterhus?= <duesterhus@woltlab.com>
Date: Tue, 18 Aug 2020 14:51:58 +0200
Subject: [PATCH] Use proper import for Ui/Screen

---
 .../files/js/WoltLabSuite/Core/Ui/Reaction/Handler.js       | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Reaction/Handler.js b/wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Reaction/Handler.js
index 5aad47ce2..75f66fa6b 100644
--- a/wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Reaction/Handler.js
+++ b/wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Reaction/Handler.js
@@ -12,19 +12,17 @@ define(
 		'Ajax',      'Core',                            'Dictionary',           'Language',
 		'ObjectMap', 'StringUtil',                      'Dom/ChangeListener',   'Dom/Util',
 		'Ui/Dialog', 'WoltLabSuite/Core/Ui/User/List',  'User',                 'WoltLabSuite/Core/Ui/Reaction/CountButtons',
-		'Ui/Alignment', 'Ui/CloseOverlay'
+		'Ui/Alignment', 'Ui/CloseOverlay', 'Ui/Screen'
 	],
 	function(
 		Ajax,        Core,              Dictionary,             Language,
 		ObjectMap,   StringUtil,        DomChangeListener,      DomUtil,
 		UiDialog,    UiUserList,        User,                   CountButtons,
-		UiAlignment, UiCloseOverlay
+		UiAlignment, UiCloseOverlay, UiScreen
 	)
 	{
 		"use strict";
 		
-		var UiScreen = require('Ui/Screen');
-		
 		/**
 		 * @constructor
 		 */
-- 
2.28.0
```

It worked fine without any JavaScript errors.

FWIW: There are quite a few obsolete imports that can (and should) be removed.

![image](https://user-images.githubusercontent.com/209270/90515285-b3ee3980-e162-11ea-8288-bab8d65b70be.png)